### PR TITLE
Baseline done

### PR DESCRIPTION
**Pull request by nlcurry**
Spec file made for Sales class

### CODE REVIEW

File: lib/markets.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Markets
```
Comment (jnf) [inline lib/markets.rb]: These data classes need to be defined as belonging to the `FarMar` module, like this:

``` ruby
class FarMar::Market

end
```

**Comment by jnf** [comment]
Hi! There's a couple of things I'd like you to update before you move on:
1. The standard in Ruby is for class names to be singular, i.e., `class FarMar::Market` rather than `class Markets`. This is because each _instance_ of the class represents a single object.
2. All of the data classes should be defined as belonging to the module. See the comment above for an example.

Let me know when you've made these changes and I'll take another look.

**Comment by nlcurry** [comment]
Fixed baseline is online.

**Comment by jnf** [comment]
Sorry to be so nitpicky, but the convention in Ruby is for class names to be singular, i.e., `class FarMar::Market` rather than `class Farmar::Markets`. This is because each _instance_ represents a single object. Please take a moment to rename the files and classes to be singular.

**Comment by nlcurry** [comment]
Got it. Sorry, I missed that.

**Comment by jnf** [comment]
Awesome, thanks. :+1:

**Pull request closed without merging**